### PR TITLE
Adding Keycloak 

### PR DESCRIPTION
**Pull request by harsh4870**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [x] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [ ] Tested that the application works with the proposed updates applied (including a screenshot)

### CODE REVIEW

**Review by saiyam1814 — CHANGES_REQUESTED** [review]
Thank you for the PR , 
can you include the required services section in the manifest.yaml file

https://github.com/civo/kubernetes-marketplace/blob/master/traefik2-loadbalancer/manifest.yaml

Also why you are using a Loadbalancer when you have provided the ingress in post install? 
Can you please include the screenshots in the comments and show how its working.

**Comment by saiyam1814** [comment]
ping @harsh4870 ^^

**Comment by harsh4870** [comment]
@saiyam1814  Hi Saiyam 

thanks for reviewing and sharing suggestions, sure will update the screenshots in the comment with details. 

Kept load balancer in service if anyone wants to access directly over the IP or else can use the Ingress as per need, added ingress for ref.

**Comment by saiyam1814** [comment]
> @saiyam1814 Hi Saiyam
> 
> thanks for reviewing and sharing suggestions, sure will update the screenshots in the comment with details.
> 
> Kept load balancer in service if anyone wants to access directly over the IP or else can use the Ingress as per need, added ingress for ref.

Hey @harsh4870 any updates with the screenshots working using the steps that you mentioned in the files in the PR ?

**Comment by harsh4870** [comment]
> On it, will be updating it before EOD sorry for the delay.

**Comment by saiyam1814** [comment]
HEllo @harsh4870 left a comment for post install. Once you update please ping and I will merge

File: keycloak/post_install.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Keycloak
```
[the remark this answers is not in the document]
Comment (harsh4870) [reply keycloak/post_install.md:1]: Sure, i have updated the documentation.

**Comment by harsh4870** [comment]
@saiyam1814  Sure, thanks, i have added the note and updated the doc.

**Comment by saiyam1814** [comment]
LGTM the changes will be live in 15-20 minutes, please do test tin production as well.

**Comment by saiyam1814** [comment]
@harsh4870 how do you get the creds for the admin console? anything pointed in the docs can we have that part of post install ?

**Comment by harsh4870** [comment]
@saiyam1814  Default id password is admin, admin for keycloak. One we are setting into deployment env. 

```
env:
        - name: KEYCLOAK_ADMIN
          value: "admin"
        - name: KEYCLOAK_ADMIN_PASSWORD
          value: "admin"
```

**Comment by saiyam1814** [comment]
@harsh4870 you could have used the variables to autogrnerate the passwords. 
Also can you fix post install 
```
[http://selenium/.<your-cluster-id>.[k8s.civo.com](http://k8s.civo.com/)](https://www.keycloak.org/getting-started/getting-started-kube)
```

**Comment by harsh4870** [comment]
@saiyam1814  Raised https://github.com/civo/kubernetes-marketplace/pull/340

**Pull request merged**